users/service.go

```diff
@@ -215,12 +245,14 @@ func (svc usersService) User(ctx context.Context, token string) (User, error) {
 	}, nil
 }
 
-func (svc usersService) ListUsers(ctx context.Context, token string, groupID string, offset, limit uint64, um Metadata) (UserPage, error) {
+func (svc usersService) ListUsers(ctx context.Context, token string, offset, limit uint64, email string, m Metadata) (UserPage, error) {
+	// TODO: Check Policy
```

Comment:
Same here. No need to pen two issues, just reference the methods that need to be updated.

Reply:
Here it is: https://github.com/mainflux/mainflux/issues/1267